frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -320,7 +320,7 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
                 <FileIcon>
                   <FaFileExcel
                     size={36}
-                    title="Export to Excel"
+                    data-testid="excel-icon"
```

Comment:
Interesting, is this something that gets stripped by the transpiler for the production build, or is it just a simple way to use `queryByTestId(...)`?

Reply:
I have no idea about the first part, but I added it for ease of testing